src/fides/api/service/connectors/consent_email_connector.py

```diff
@@ -230,12 +232,12 @@ def batch_email_send(self, privacy_requests: Query) -> None:
                 Consent(**pref) for pref in privacy_request.consent_preferences or []
             ]
 
-            # Privacy preferences for new workflow
             filtered_privacy_preference_records: List[
-                PrivacyPreferenceHistory
-            ] = filter_privacy_preferences_for_propagation(
-                self.configuration.system, privacy_request.privacy_preferences
+                PrivacyPreferenceHistoryV2
+            ] = filter_privacy_preferences_for_propagation(  # type: ignore[assignment]
+                self.configuration.system, privacy_request.privacy_preferences_v2
             )
+
```

Comment:
Type just changing related to how Privacy Requests have a relationship to the new Privacy Preference V2 table, that is the record that stores the consent details we need to propagate to third parties.

Reply:
sure, I thought the diff would be more confusing though since basically none of the logic in the original preferences model holds/original endpoints hold!

Reply:
that's a fair point! 🤔

Reply:
but for this file in particular, I'm looking into your advice of leaving the historical models in place, which will remove the need for this update.  I think I will keep the "updated" historical models in new files though.